common/src/IO/EntityDefinitionClassInfo.cpp

```diff
@@ -82,7 +83,7 @@ namespace TrenchBroom {
         }
 
         std::vector<std::shared_ptr<Assets::AttributeDefinition>> EntityDefinitionClassInfo::attributeList() const {
-            return MapUtils::valueList(m_attributes);
+            return kdl::values(m_attributes);
```

Comment:
kdl::map_values for readibility?

Reply:
Good point, but please Review the CollectionUtils stuff in the other PR. This is wip and only includes that stuff by necessity.

Reply:
changed on  #2847